src/components/data.json

```diff
@@ -0,0 +1,102 @@
+ 
```

Comment:
use fetch for getting data from json, not json loader

Reply:
added fetch